### DISCUSSION TITLE
Autocomplete not working

### DISCUSSION THREAD

**Discussion by emailman** [pybricks/support#1258] [Q&A]
I am running the current version of PyBricks on Windows 10 and autocomplete works as expected.  I'm tutoring someone who installed PyBricks on MacOS and autocomplete is not working.  Any ideas about how to troubleshoot this or how to fix it for my student?

**Comment by dlech** [comment]
There seems to be some problem with our web server. Someone else mentioned this not working recently too. I can't find the other issue to link it though.
